lib/rag/layer/requirements.txt

```diff
@@ -2,6 +2,7 @@ opensearch-py
 requests-aws4auth
 PyPDF2==3.0.1
 langchain==0.1.0
+langchain-openai==0.0.6
```

Comment:
Just a heads up that we may want to look into updating these since they're a bit old and the OpenAIEmbeddings client is on a deprecation path. Not a blocker for now (especially since I'm the one that committed that), but will possibly do a pass before official release to ensure we're not too old on requirements versions.